### ISSUE TITLE
Ask: how about translations

### ISSUE THREAD

**Issue by jiyinyiyong** [haskell/haskell-wiki-configuration#27]
I'm not sure but maybe I could translate several of the posts into Chinese for learning purpose. Do you have any source code for the Wiki posts? Markdown or something? It's not hosted on GitHub so no easy way to grab.

The page I'm particularly interested: http://wiki.haskell.org/IO_inside .

**Comment by Anton-Latukha** [comment]
The source code of wiki articles is hosted on the wiki.

There is an `Edit` tab on all the wikis.

MediaWiki engine (classical Wikis, as Wikipedia, Haskell Wiki), use Wiki markup syntax.

If you would like, there should be converters between Markdown and Wiki markup.

**Comment by jiyinyiyong** [comment]
Login system is quite counterintuitive... now I finally managed to log in and got the text. Thanks for the hint. Luckily the syntax is still similar to Markdown.

**Comment by Anton-Latukha** [comment]
First Wiki is WikiWikiWeb - 1994.
MediaWiki & Wikipedia - 2001.
Git was invented in - 2005.

And Wikis were never created to manage them through Git, the whole idea of them is that any people without the need of technical knowledge can edit the pages.
Hawaiian *wiki wiki* is - ‘very quick’, then the meaning of the *WikiWikiWeb* speaks for itself.

**Comment by jiyinyiyong** [comment]
Long history of Haskell!

I still prefer using GitHub when I want to maintain things.

**Comment by Anton-Latukha** [comment]
`GitHub` was founded in 2008.
Haskell founded in 1990. So yes, long history of Haskell.
But the Haskell Wiki is actually convenient place and resource.

The wiki engine `GitHub` provides - is laughable at its very best. GitHub wikis even do not allow to add pictures to them, the exploit hack needs to be used to trick GitHub into adding pictures there, if the page is renamed - all links to that page are broken, there are no transition possibility, I can keep counting the essentials lacking in GitHub wiki engine.

It is impossible to host a real, big, convenient, useful wiki on GitHub.

**Comment by jiyinyiyong** [comment]
Yes GitHub has those problems. Actually I was thinking about using GitHub to maintain source code of the side https://github.com/reactjs/reactjs.org , rather than using  GitHub Wiki.

Never mind... I have no indention to complain about the Wiki Haskell.org uses. Just it kept telling me I failed to log in and I was confused for many minutes. Being a Chinese developer using ClojureScript and TypeScript, I guess I would never contribute to English version of Haskell wiki.
